src/main/java/seedu/agendum/logic/commands/ScheduleCommand.java

```diff
@@ -9,7 +9,6 @@
 import seedu.agendum.model.task.*;
 import seedu.agendum.model.task.UniqueTaskList.TaskNotFoundException;
 
-//@@author A0003878Y
```

Comment:
I was thinking if we should claim the entire class with header comments and the constants. (then it will be weird without the closing brackets)
should i change mine back?